Minio/ApiEndpoints/ObjectOperations.cs

```diff
@@ -1195,18 +1197,21 @@ private async Task RemoveUploadAsync(RemoveUploadArgs args, CancellationToken ca
         var etags = new Dictionary<int, string>();
         for (partNumber = 1; partNumber <= partCount; partNumber++)
         {
-            var dataToCopy = await ReadFullAsync(args.ObjectStreamData, (int)partSize).ConfigureAwait(false);
-            if (dataToCopy == null && numPartsUploaded > 0) break;
+            var shared = ArrayPool<byte>.Shared;
```

Comment:
but the block below in a `try { } finally { ArrayPool<byte>.Shared.Return(shared); }` to avoid memory leak if an exception is thrown.

Reply:
Yes, agreed.
That is one of the incoming changes.